### PR TITLE
Kl list indexes

### PR DESCRIPTION
**Pull request by kl15115**
image

### CODE REVIEW

**Comment by ekehoe32** [comment]
@kl15115 Looks like you need to: npm install encoding

**Comment by kl15115** [comment]
Added the encoding not sure what it is failing at now

File: client/src/app/register/page.tsx
Comment (JasonAizkalns) [inline client/src/app/register/page.tsx:1]: should we do className="... max-w-md" instead? keeps it responsive. see: https://tailwindcss.com/docs/max-width

Comment (ekehoe32) [reply client/src/app/register/page.tsx:1]: Yes! Let's try to adhere to our stack and use tailwind/daisyui for all css.

Comment (ekehoe32) [reply client/src/app/register/page.tsx:1]: Added PINECONE_ENV and PINECONE_API_KEY, build now succeeds.

Comment (ekehoe32) [reply client/src/app/register/page.tsx:1]: Added PINECONE_ENV and PINECONE_API_KEY, build now succeeds.

**Comment by ekehoe32** [comment]
> Added the encoding not sure what it is failing at now

Added PINECONE_ENV and PINECONE_API_KEY, build now succeeds.

**Pull request merged**